### PR TITLE
feat: init vuln-to-kusto step

### PR DESCRIPTION
**Pull request by bcho**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:

```
none
```

### CODE REVIEW

File: .pipelines/templates/.builder-release-template.yaml [hunk 1]
```diff
@@ -170,6 +170,19 @@ steps:
...
         echo "##vso[task.setvariable variable=IS_NOT_1804]${IS_NOT_1804}"
     displayName: 'Check if OS version is not 18.04'
+  - task: AzureCLI@2
+    inputs:
+      azureSubscription: agentbaker-vuln-to-kusto
+      scriptType: 'bash'
+      scriptLocation: 'inlineScript'
+      inlineScript: |
+        export AZCOPY_AUTO_LOGIN_TYPE=AZCLI
+        azcopy copy https://vulntokustobinary.blob.core.windows.net/vuln-to-kusto/v0.0.1-ee36fabc1ec4f415a022042ea89f751bf05da6ef/vuln-to-kusto-vhd_Linux_x86_64 ./vuln-to-kusto
```
Comment (cameronmeissner) [inline .pipelines/templates/.builder-release-template.yaml:180]: assuming this storage is locked down? does that mean we're already able to pull from it using the agent identity?

Comment (bcho) [reply .pipelines/templates/.builder-release-template.yaml:180]: yep the storage account is private and configured to be accessible with the identity used in this step (`agentbaker-vuln-to-kusto`)

**Review by cameronmeissner — APPROVED** [review]
lgtm for now, feel free to ping me for a further review if you need it

**Comment by cameronmeissner** [comment]
closing this since I believe the work @jason1028kr did in a recent PR takes care of this

**Pull request closed without merging**